fortinet_fortigate/help.md

```diff
@@ -274,13 +274,18 @@ This action is used to create an address object.
 
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
-|cidr|integer|32|True|CIDR|None|None|
-|ip|string|None|True|IP|None|None|
-|name|string|None|False|Optional name to give this address object. If not provided, the name will be the IP address|None|None|
+|host|string|None|True|This can be an IP-Address, an IP CIDR or a domain name. e.g. 192.168.1.1, 192.168.1.0/24, google.com|None|198.51.100.100|
```

Comment:
```suggestion
|host|string|None|True|This can be an IP-Address, an IP CIDR or a domain name|None|198.51.100.100|
```

Reply:
I think this is fine, original has more detail